transhumandesign/kag-base
# Fireplace emits sound from the start and doesn't spin

## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

Fireplace had an issue where the CampfireSound.ogg didn't play unless you extinguished and reignited the fireplace.
Also fireplace could spin around if falling on ledges and fire burning downwards looks odd.
This change makes it so Fireplace will always play the campfire sound when it is on fire and will not spin around anymore.

## Steps to Test or Reproduce

- Go to solo mode "Save the princess" and notice the campfire at the beginning is emitting sound correctly.
- Go to Sandbox and spawn Fireplaces, notice they emit sound correctly.
- Spawn a fireplace above a ledge, notice it doesn't spin around anymore.
